tutorials/2.6/window-events.php

```diff
@@ -161,23 +161,38 @@ class. Only these two functions can produce valid events, any attempt to use an
     <a class="internal" href="./window-inputs.php" title="Real-time inputs tutorial">dedicated tutorial</a>).
 </p>
 <p>
-    The member associated with these events is <code>event.key</code>, it contains the code of the pressed/released key, as well as the current state of
+    The member associated with these events is <code>event.key</code>, it contains the scancode and key code of the pressed/released key, as well as the current state of
     the modifier keys (alt, control, shift, system).
 </p>
+<p>
+    Scancodes are unique values for each physical key on a keyboard, regardless of the language or layout. Key codes represent keys based on the user's chosen layout.
```

Comment:
I find it easier to understand this way:
```suggestion
    Scancodes are unique values for each physical key on a keyboard, regardless of the language or layout, whereas key codes represent keys based on the user's chosen layout.
```